src/components/search/Search.jsx

```diff
@@ -42,6 +42,7 @@ const Search = () => {
         indexName={config.ALGOLIA_INDEX_NAME}
         searchClient={algolia.client}
       >
+        <Configure facetingAfterDistinct filters={filters} />
```

Comment:
I think we should do this in frontend-enterprise repo. Since `SearchHeader` is imported from there and being used at some other places as well so this count issue will remain as it is at other places.

Reply:
Where exactly would this configuration go in `frontend-enterprise`? Isn't the <Configure> component supposed to be wrapped inside <InstantSearch>?